src/components/page.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+import Counter from './counter';
+import Buttons from './buttons';
+
+function Page() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
+  function handleIncreaseByNumber(number) {
+    setState({
+      count: count + number,
+    });
+  }
+
+  return (
+    <>
+      <Counter count={count} onIncreaseByNumber={handleIncreaseByNumber} />
```

Comment:
```suggestion
      <Counter count={count} onClick={handleIncreaseByNumber} />
```

만약에 저라면 이렇게 작성했을 것 같아요.
컴포넌트의 Props 의 이름과 핸들러 함수의 이름은 해당 이름이 적절해서 '우연히'도 같을 수는 있어도 각각 별개라고 생각하시면 좋아요.
각 컴포넌트입장에서 어떤 의미를 가지는지에 따라 이름을 결정하시면 좋을 것 같아요!